template.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"strings"
+	"text/template"
+
+	"github.com/pkg/errors"
+)
+
+type BotMessageData struct {
+	Template                string
+	Branch                  string
+	AutoReleaseEnvironments []string
+}
+
+func BotMessage(data BotMessageData) (string, error) {
+	var message strings.Builder
+
+	template := template.New("test")
+	template, err := template.Parse(data.Template)
+	if err != nil {
+		return "", errors.Wrapf(err, "invalid template: %s", data.Template)
+		// or "parsing template"
```

Comment:
Parsing template would, to me, be the proper error context, as we don't know if it's invalid or some other error happened.

```suggestion
		return "", errors.Wrapf(err, "parsing template: '%s'", data.Template)
```